.gitlab-ci.yml

```diff
@@ -20,7 +20,7 @@ variables:
     - git remote add -f origin https://github.com/EducationalTestingService/rsmtool.git
     - git config core.sparsecheckout true
     - echo "tests/*" > .git/info/sparse-checkout
-    - git pull --depth=1 origin main
+    - git pull --depth=1 origin release/12.0.0
```

Comment:
No, it's because we usually make sure all the test data fixes are already in main by the time we are releasing but this time I just did the release branch directly :)

I will make sure not to merge it this way.